pom.xml

```diff
@@ -86,7 +86,6 @@
                         <artifactId>nexus-staging-maven-plugin</artifactId>
                         <version>1.6.8</version>
                         <extensions>true</extensions>
-                        <inherited>false</inherited>
```

Comment:
Nexus was not publishing any artifacts for conduit-bom or conduit itself. Only conduit-parent.

See https://oss.sonatype.org/#view-repositories;public~browsestorage~io/rtr.